.github/scripts/tests/tests.sh

```diff
@@ -202,7 +223,6 @@ run_kind_tests() {
   apply_pip_server_configmap
   run_tests
   run_tests_dspa_external_connections
-  clean_up
 }
 # ------------------------------------
```

Comment:
nit: these comments here look messy, can you get rid of these `# ----` lines? https://github.com/opendatahub-io/data-science-pipelines-operator/blob/f3871a3dcf01b151e4a849ff062f05a1c9a0ef77/.github/scripts/tests/tests.sh#L227-L243

Reply:
For me, after spending some time looking at the source, it seems easy to understand the separation of responsibilities.

Reply:
disagree, I think it's fairly common to have opts handling at the bottom or top in a bash script, I find the comments contribute to clutter, we can simply just leave the "# Run" comment, or expand on it like "# Script Entry point", or wrap it in a main()

Reply:
OK. going to not fight by that if you have a strong opition.

Reply:
Done